build/root/Makefile

```diff
@@ -623,4 +623,5 @@ bazel-release:
 else
 bazel-release:
 	bazel build //build/release-tars
+	bazel shutdown
```

Comment:
should we do the same thing after `bazel build -- //... -//vendor/...` in the `bazel-build:` target, or is there a reason just to do it here?

Reply:
@liggitt i don't see evidence of any CI jobs that fail during that step. So i don't want to add it anywhere else. We have clear evidence in the logs that the next bazel call after `build //build/release-tars` fails, so just adding it here.

Reply:
ok

Reply:
we're going to start it again for push-build, this will cost us some time.